tests/10apidoc/44account_data.pl

```diff
@@ -41,6 +42,42 @@ sub matrix_add_room_account_data
    );
 }
 
+=head2 matrix_get_account_data
+
+   matrix_get_account_data( $user, $type, $content )->get;
```

Comment:
why does this have a `->get`?

Reply:
copy/paste. I guess it's not needed?

Reply:
looks like the other functions here also have it...

Reply:
Github lost my comment that I noticed that :(

I have no idea what its doing with its life